src/api/procedures/quitCustody.ts

```diff
@@ -32,7 +32,8 @@ export async function prepareQuitCustody(
 
   const { portfolio } = args;
 
-  const isOwnedBy = await portfolio.isOwnedBy();
+  const signer = await context.getSigningIdentity();
```

Comment:
I think a unit test is a bit hard for this. I found it with an integration one, so we'll have that coverage though.

The other solution is to destructure the given Portfolio and create a new one with the current context.

Its very edge case since you need to be a custodian that somehow have a key ring with your customer's private key as the default in your signing manager

Reply:
Can we atleast assert that isOwnedBy was called with {identity: signer} param?